qemu/tests/cfg/viostor_in_use.cfg

```diff
@@ -0,0 +1,62 @@
+- viostor_in_use:
```

Comment:
From the cfg name viostor_in_use.cfg, it should be virtio_block only, but actually it has covered virtio_scsi.
I think need update the case name.

Others look good to me.